product/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+
+# Create your models here.
+class Category(models.Model):
+    category = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'categories'
+
+class Product(models.Model):
+    category    = models.ForeignKey('Category', on_delete=models.SET_NULL, null=True)
+    name        = models.CharField(max_length=60)
+    price       = models.CharField(max_length=45)
+    description = models.TextField(max_length=2000)
```

Comment:
```suggestion
    description = models.TextField(max_length=2000)
```
총 문자열 bytes 2000을 초과하지 않는다면, 그냥  CharFiled로 해도 될 것 같습니다. 4000 초과하는 부분에 대해 적용해주세요 .

Reply:
이후의 코드에 비슷한것들이 자주 보이는데 적용 부탁드려요!

Reply:
넵!!